index.js

```diff
@@ -8,8 +8,14 @@ module.exports = {
     },
   },
   included() {
-    this.app.import('node_modules/shepherd.js/dist/css/shepherd.css');
+    let app = this._findHost();
 
     this._super.included.apply(this, arguments);
+
+    let addonConfig = Object.assign({}, app.options['ember-shepherd']);
+
+    if (addonConfig.includeStyles !== false) {
+      app.import('node_modules/shepherd.js/dist/css/shepherd.css');
```

Comment:
```suggestion
      this.app.import.import('node_modules/shepherd.js/dist/css/shepherd.css');
```

Should we continue using `this.app` as we were before? I'm never sure if we need `findHost` or not. We also maybe should remove this entirely and have people import the styles manually in their apps? Not sure the best future proof way to do this.